src/condor_utils/condor_event.cpp

```diff
@@ -267,14 +292,38 @@ ULogEvent::~ULogEvent (void)
 {
 }
 
+/*static*/
+ULogEventNumber ULogEvent::readEventNumber(ULogFile& ulf, char* headbuf, size_t bufsize)
+{
+	ASSERT(bufsize > 32); // header is something like: 000 (16091.000.000) 01/22 12:09:19
+	memset(headbuf, 0, 32);
```

Comment:
Is memset() supposed to be making headbuf with size 32 or is it supposed to be bufsize?